uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -62,86 +62,6 @@ $FLOAT_EQUALS_NB_DECIMALS = 4;
 
 =pod
 
-=head3 getSubjectIDs($patientName, $scannerID, $dbhr, $db)
-
-Determines the candidate ID and visit label for the subject based on patient
-name and (for calibration data) scanner ID.
-
-INPUTS:
-  - $patientName: patient name
-  - $scannerID  : scanner ID
-  - $dbhr       : database handle reference
-  - $db         : database object
-
-RETURNS: a reference to a hash containing elements including C<CandID>,
-C<visitLabel> and C<visitNo>, or, in the case of failure, C<undef>
-
-=cut
-
-sub getSubjectIDs {
```

Comment:
removed this function as we are only using the ones from the prod file, hence, we only have to maintain one instead of two